During installation, the libtool relink attempts to link to -lrpm...
The problem is that it hasn't been installed yet!  So small change causes
the libtool to instead use the build version.

Upstream-Status: Pending

Signed-off-by: Mark Hatle <mark.hatle@windriver.com>

--- rpm-5.4.0/lib/Makefile.am.orig
+++ rpm-5.4.0/lib/Makefile.am
@@ -90,6 +90,9 @@
 libsql_la_SOURCES	= libsql.c
 libsql_la_LIBADD	= librpm.la $(RPMDB_LDADD_COMMON)
 
+# pkglib libraries needs to have usrlib libraries already installed!
+install-pkglibLTLIBRARIES: install-usrlibLTLIBRARIES
+
 install-data-hook:
 if !ENABLE_BUILD_LAFILES
 	-for l in $(usrlib_LTLIBRARIES); do \
